engine/verification/fetcher/requester.go

```diff
@@ -0,0 +1,22 @@
+package fetcher
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// ChunkDataPackRequester encapsulates the logic of requesting a chunk data pack from an execution node.
+type ChunkDataPackRequester interface {
```

Comment:
Noting that we have a [generic entity requester/provider engine](https://github.com/onflow/flow-go/tree/7e529708e81f9fa36e61b7177b65939a1b0a1e75/engine/common/requester) - we should re-use that here if at all possible.